test/fixtures/payments-fixtures.js

```diff
@@ -0,0 +1,20 @@
+'use strict'
+// Create random values if none provided
+const randomExternalId = () => Math.random().toString(36).substring(7)
+const randomAmount = () => Math.round(Math.random() * 10000) + 1
+
+module.exports = {
+  validTokenExchangeResponse: (opts = {}) => {
+    const data = {
+      external_id: opts.external_id || randomExternalId(),
+      amount: opts.amount || 'pay-api-token',
+      name: opts.type || 'CHARGE',
+      price: opts.state || randomAmount()
+    }
+    return {
+      getPlain: () => {
+        return data
+      }
+    }
+  }
+}
```

Comment:
Maybe it’s fine but if this file is only ever going to export one function maybe it doesn’t need to be named?

Could be just 
``` js
module.exports = (opts = {}) => {
...
}
```

Reply:
This will export also the pactified response - I'd keep it like this for now, I didn't add pact because it would've probably been too much work for this story

Reply:
Ah cool